### PR TITLE
Rework "Creating your first chart"

### PR DESCRIPTION
**Pull request by ThatZeoMan**
NOTES:
- BPM section is currently unfinished. It'll take a while for me to condense down the stuff in /docs-tips/music/2_bpm.md
- I did this PR entirely on GitHub Web, so I can't move files. Someone'll have to go in and move images & change references
- I also didn't delete the old offset document. Someone should probably do that.

### CODE REVIEW

File: docs/4_first-chart.md [hunk 1]
```diff
@@ -7,61 +7,3 @@ import ReactPlayer from 'react-player'
 # Making your first remix with Heaven Studio
```
Comment (patataofcourse) [inline docs/4_first-chart.md:7]: what's the point of this file still existing???

File: docs/firstRemix/1_song-offset.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+---
```
Comment (patataofcourse) [inline docs/firstRemix/1_song-offset.md:1]: no need to duplicate this - simply link to the file in docs-tips/music/1_offset.md in an introductory file to this category

**Review by patataofcourse — CHANGES_REQUESTED** [review]
Here's some organization stuff

File: docs/firstRemix/1_song-offset.md (hunk 2 above)
Comment (ThatZeoMan) [reply docs/firstRemix/1_song-offset.md:1]: i wanted to _move_ it, but i'm on github web and can't do that. i think it definitely belongs natively in the first remix, as its one of if not the most important parts of making a remix, and its very easy to mess up

File: docs/4_first-chart.md (hunk 1 above)
Comment (ThatZeoMan) [reply docs/4_first-chart.md:7]: good idea lol

**Comment by ThatZeoMan** [comment]
@Saraistupid aight, a few organization changes have been made

**Comment by Saraistupid** [comment]
Merging for now, we'll discuss everything later.

**Pull request merged**